.github/workflows/release.yaml

```
@@ -262,103 +257,67 @@ jobs:
       - name: Install Rust toolchain
         uses: dtolnay/rust-toolchain@stable
         with:
-          targets: ${{ env.TARGET }}
-
-      - name: Build binary
-        run: |
-          cargo build --profile="${{ env.PROFILE }}" --target="${{ env.TARGET }}" --no-default-features
-
-      - name: Prepare package
-        run: |
-          mkdir -p "${{ env.OUT_DIR }}"
-          cp "target/${{ env.TARGET }}/${{ env.PROFILE }}/${{ env.BINARY }}" "${{ env.OUT_DIR }}"
-          cp -r assets "${{ env.OUT_DIR }}" || true # Ignore error if `assets` folder does not exist
-
-      - name: Compress package
-        working-directory: ./build/${{ env.PACKAGE }}
+            targets: aarch64-apple-darwin
+      - name: Build release for Apple Silicon
         run: |
-          hdiutil create -fs HFS+ -volname "${{ env.PACKAGE_NAME }}" -srcfolder "${{ env.PACKAGE_NAME }}.app" "../${{ env.PACKAGE }}.dmg"
-
-      - name: Upload package to artifacts
-        uses: actions/upload-artifact@v4
-        with:
-          path: build/${{ env.PACKAGE }}.dmg
-          name: ${{ env.PLATFORM }}
-          retention-days: 1
-
-      - name: Upload package to Github release
-        if: ${{ env.UPLOAD_PACKAGES_TO_GITHUB_RELEASE == 'true' }}
-        uses: svenstaro/upload-release-action@v2
+          SDKROOT=$(xcrun -sdk macosx --show-sdk-path) cargo build --profile="${{ env.PROFILE }}" --no-default-features --target=aarch64-apple-darwin
+      - name: Install rust toolchain for Apple x86
+        uses: dtolnay/rust-toolchain@master
```

```suggestion
        uses: dtolnay/rust-toolchain@stable
```
Unless there's a reason for this?